build.html

```diff
@@ -266,6 +280,23 @@ <h2>
           </div>
         </div>        
       </div>
+
+      <div id="bulk-best-practices" class="guide-content">
+        <h1>
+          Best Practices for Utilizing BCDA
+        </h1>
+        <div class="ds-u-font-size--base ">
+          {% include build/best_practices.html %}
+        </div>
```

Comment:
Suggestion to remove the best practices description
```suggestion

```

Reply:
updated!